pyproject.toml

```diff
@@ -97,8 +97,13 @@ exclude = '''
 
 [tool.ruff]
```

Comment:
Good to see. On other project I have in fact replaced isort and black with ruff as well.